svir/dialogs/ipt_dialog.py

```diff
@@ -74,6 +74,19 @@ class IptPythonApi(GemApi):
     (other shared methods are defined in the CommonApi)
     """
 
+    @pyqtSlot(result=bool)
+    def on_same_fs(self):
+        """
+        Check if the engine server has access to the ipt_dir
+        """
+        checksum_file_path, local_checksum = \
+            self.parent().irmt.get_ipt_checksum()
+        self.parent().irmt.drive_oq_engine_server(show=False)
+        on_same_fs = self.parent().irmt.drive_oq_engine_server_dlg.on_same_fs(
```

Comment:
Please add a `on_same_fs` method to `irmt` and, inside it, run `drive_oq_engine_server()` initialization and the private `drive_oq_engine_server_dlg.on_same_fs()` method.